### ISSUE TITLE
Appium 1.6.5 - iOS 10.3.3 Real Device - Safari - Cant click on a button that opens up a new link

### ISSUE THREAD

**Issue by mirza-ae** [appium/appium#9015]
## The problem

Mouse clicks for a particular button doesnt work for Safari browsers. Appium can detect the element fine, but when it clicks on it, nothing happens. Clicking on the button manually works. This issue cant be replicated on other browsers.

## Environment

* Appium version (or git revision) that exhibits the issue: 1.6.5
* Desktop OS/version used to run Appium: Mac OSX Sierra
* Node.js version (unless using Appium.app|exe): 8.2.1
* Mobile platform/version under test: iOS Safari
* Real device or emulator/simulator: iPhone 7 Plus Real device

## Details

1. Access the site, https://www.aetv.com/profile/sign-in?mode=register&referrerUrl=http://www.aetv.com using an iOS Safari browser.
2. Using a locator, click on the 'SIGN UP WITH FACEBOOK' button. 
3. Observe that nothing happens.

## Link to Appium logs
https://gist.github.com/mirza-ae/a755d14aff010bced6d08f666b550b50

## Code To Reproduce Issue [ Good To Have ]
`
By xpath = By.xpath("//*[text()='Sign up with Facebook']");
boolean isDisplayed = driver.findElement(xpath).isDisplayed();
boolean isEnabled = driver.findElement(xpath).isEnabled();
int count = driver.findElements(xpath).size();
driver.findElement(xpath).click();
`

**Comment by imurchie** [comment]
Try setting the `nativeWebTap` desired capability to `true`.

**Comment by mirza-ae** [comment]
@imurchie 

I thought of doing that, but the documentation states that it's for iOS Simulators only.
https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md

This use case is for real devices.

**Comment by imurchie** [comment]
Ah, yes, until the most recent beta it is only on simulators. It works with real devices on 1.6.6-beta.4

**Comment by mirza-ae** [comment]
@imurchie 

I updated appium to the beta version, and set "nativeWebTap" to true in the capabilities, and that fixed the issue.

Do you happen to know when this is going to be added to the final version?

**Comment by mykola-mokhnach** [comment]
It's ongoing (see https://github.com/appium/appium/issues/8949)

**Comment by mike-malandra** [comment]
When I click a link in the webview context, a new window is not opening. 

I am using the dotnet version of appium on Browserstack with a real device. I have tried both version 1.7.0 and 1.7.1 of appium. I have set nativeWebTap to true in the capabilities. 

I am not sure if this a Browserstack issue or an appium issue. Has anyone used the nativeWebTap to open a new context successfully in 1.7.0+?
